lib/beaker/host/unix.rb

```diff
@@ -64,16 +64,16 @@ def self.aio_defaults
       h = self.foss_defaults
       h['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
       h['puppetservice']  = 'puppetserver'
-      h['puppetbindir']   = '/opt/puppetlabs/agent/bin'
+      h['puppetbindir']   = '/opt/puppetlabs/puppet/bin'
```

Comment:
@kevpl we had a discussion about this, and we want beaker to be using the "public" AIO bin directory which is `/opt/puppetlabs/bin`.

There are some puppet acceptance tests, e.g. https://github.com/puppetlabs/puppet/blob/master/acceptance/tests/environment/use_enc_environment.rb#L18, that assume `puppetbindir` points to the "private" bin directory, which contains ruby. We are going to update the tests on our side, but don't let that block beaker.